### PR TITLE
ticdc: update changefeed scheduler configs

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Update changefeed scheduler configs.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13341
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -91,11 +91,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # Ignore inser
...
 [scheduler]
 # Splits a table into multiple replication ranges based on the number of Regions, and these ranges can be replicated by multiple TiCDC nodes.
-# Note:
-# 1. Scaling out a single large table is an experimental feature. It is not recommended that you use it in the production environment.
-# 2. This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
-# 3. TiCDC does not split tables with fewer Regions than this parameter value into multiple replication ranges.
-# region-per-span = 50000
+# Note: This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
+# The value is "false" by default. Set it to "true" to enable it.
+# enable-table-across-nodes = true
```
Comment (overvenus) [inline ticdc/ticdc-changefeed-config.md:96]:
```suggestion
enable-table-across-nodes = true
```

**Review by overvenus — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -91,11 +91,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # Ignore inser
...
 [scheduler]
 # Splits a table into multiple replication ranges based on the number of Regions, and these ranges can be replicated by multiple TiCDC nodes.
-# Note:
-# 1. Scaling out a single large table is an experimental feature. It is not recommended that you use it in the production environment.
-# 2. This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
-# 3. TiCDC does not split tables with fewer Regions than this parameter value into multiple replication ranges.
-# region-per-span = 50000
+# Note: This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
+# The value is "false" by default. Set it to "true" to enable it.
+enable-table-across-nodes = true
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:96]:
```suggestion
enable-table-across-nodes = false
```

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -91,11 +91,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # Ignore inser
...
+# Enable this feature on a table with a minimum number of Regions to be included.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:97]: 这里我修改了下中文，请看看是否合适

`# 打开该功能后，该功能只对 Region 个数大于等于 `region-threshold` 值的表生效。`

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -243,8 +243,7 @@ This feature splits a single large table into multiple data ranges based on the
 
 > **Warning:**
 >
-> - Scaling out a single large table is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
-> - TiCDC v6.6.0 only supports scaling out a large single table on Kafka changefeeds.
+> - TiCDC v7.0 only supports scaling out a large single table on Kafka changefeeds.
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:246]:
```suggestion
> TiCDC v7.0.0 only supports scaling out a large single table on Kafka changefeeds.
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -243,8 +243,7 @@ This feature splits a single large table into multiple data ranges based on the
...
 
 Sample configuration:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:248]: 这里中文也修改了，英文需要对应改下
image

**Review by overvenus — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -91,11 +91,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # Ignore inser
 
 [scheduler]
 # Splits a table into multiple replication ranges based on the number of Regions, and these ranges can be replicated by multiple TiCDC nodes.
-# Note:
-# 1. Scaling out a single large table is an experimental feature. It is not recommended that you use it in the production environment.
-# 2. This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
-# 3. TiCDC does not split tables with fewer Regions than this parameter value into multiple replication ranges.
-# region-per-span = 50000
+# Note: This parameter only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
+# The value is "false" by default. Set it to "true" to enable it.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:95]:
```suggestion
# The value is "false" by default. Set it to "true" to enable this feature.
```

File: ticdc/ticdc-changefeed-config.md [hunk 7]
```diff
@@ -91,11 +91,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # Ignore inser
...
+enable-table-across-nodes = false
+# When you enable this feature, it only takes effect for tables with the number of regions greater than or equal to the `region-threshold` value.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:97]:
```suggestion
# When you enable this feature, it only takes effect for tables with the number of regions greater than the `region-threshold` value.
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 8]
```diff
@@ -243,12 +243,20 @@ This feature splits a single large table into multiple data ranges based on the
 
 > **Warning:**
 >
-> - Scaling out a single large table is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
-> - TiCDC v6.6.0 only supports scaling out a large single table on Kafka changefeeds.
+> TiCDC v7.0.0 only supports scaling out a large single table on Kafka changefeeds.
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:246]:
```suggestion
> TiCDC v7.0.0 only supports scaling out the load of a large single table on Kafka changefeeds.
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 9]
```diff
@@ -243,12 +243,20 @@ This feature splits a single large table into multiple data ranges based on the
...
 
 Sample configuration:
 
 ```toml
 [scheduler]
-region-per-span = 50000
+# Set it to "true" to enable this feature.
+enable-table-across-nodes = true
+# When you enable this feature, it only takes effect for tables with the number of regions greater than or equal to the `region-threshold` value.
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:254]:
```suggestion
# When you enable this feature, it only takes effect for tables with the number of regions greater than the `region-threshold` value.
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 10]
```diff
@@ -243,12 +243,20 @@ This feature splits a single large table into multiple data ranges based on the
...
+region-threshold = 100000
+```
+
+You can query the number of Regions a table contains by the following SQL:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:258]:
```suggestion
You can query the number of Regions a table contains by the following SQL statement:
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**